## Description: Stop redundant copyright file from being installed.
## Origin/Author: Mitchell Smith <mjs@mjsprojects.net>
Index: xwax-0.9/Makefile
===================================================================
--- xwax-0.9.orig/Makefile	2011-04-16 19:19:28.000000000 +1000
+++ xwax-0.9/Makefile	2011-04-21 08:28:05.147622116 +1000
@@ -105,7 +105,6 @@
 		$(INSTALL) -m 0644 xwax.1 $(MANDIR)/man1/xwax.1
 		$(INSTALL) -d $(DOCDIR)/xwax
 		$(INSTALL) -m 0644 CHANGES $(DOCDIR)/xwax/CHANGES
-		$(INSTALL) -m 0644 COPYING $(DOCDIR)/xwax/COPYING
 		$(INSTALL) -m 0644 README $(DOCDIR)/xwax/README
 
 # Distribution archive from Git source code
